### DISCUSSION TITLE
Hall thruster wall erosion simulation

### DISCUSSION THREAD

**Discussion by yuanjiang-lance** [BLAST-WarpX/warpx#4542] [Q&A]
Hi all,

I'm quite interested in whether WarpX could simulate wall erosion in Hall thrusters or other electromagnetic thrusters. It is much related to the plasma dynamics in thruster discharge chamber.

**Comment by RemiLehe** [comment]
Thanks for the question. 
I am not very familiar with this physics. Could explain what is the physics that governs wall erosion? e.g. what is the wall typically made of, and what causes it to erode (e.g. is it because fast electron and/or ions are impacting the wall and cause sputtering of the wall material? In that case WarpX would not be able to simulate this physics, in its current state)

**Comment by aveksler1** [comment]
Off the top of my head, I know of TRI3DYN, a code that simulates dynamic wall erosion from ion irradiation. I'm not sure if the irradiation models there can represent the type of plasma you are interested in.

https://www.hzdr.de/db/Cms?pOid=64858&pNid=2689&pLang=en
